tfhe/src/integer/client_key/mod.rs

```diff
@@ -425,13 +427,24 @@ impl ClientKey {
         self.key.encrypt(message)
     }
 
+    pub fn encrypt_bool(&self, msg: bool) -> BooleanBlock {
+        BooleanBlock::new(self.encrypt_one_block(u64::from(msg)))
+    }
+
     /// Decrypts one block.
     ///
     /// This takes a shortint ciphertext as input.
     pub fn decrypt_one_block(&self, ct: &Ciphertext) -> u64 {
         self.key.decrypt(ct)
     }
 
+    pub fn decrypt_bool<Inner>(&self, ct: &BooleanValue<Inner>) -> bool
```

Comment:
Here a docstring to indicate it uses the C-like meaning for integers and bools would be nice (maybe same thing in the radix client key stuff)